src/test/java/teammates/test/pageobjects/AdminHomePage.java

```diff
@@ -95,4 +96,11 @@ public String getMessageFromResultTable(int index) {
         return getCellValueFromDataTable(index, 5);
     }
 
+    public String getJoinLink(int index) {
+        WebElement tableElement = browser.driver.findElements(By.className("table")).get(0);
+        WebElement trElement = tableElement.findElements(By.tagName("tr")).get(index);
+        WebElement tdElement = trElement.findElements(By.tagName("td")).get(5);
+        return tdElement.findElement(By.tagName("a")).getAttribute("href");
```

Comment:
I went back and forth on whether to put this in `AppPage` or `AdminHomePage` and ended up putting it in the subclass because I was thinking it was kind of a fringe case that would really only be used here. Are you thinking it might be a good resource to have in the superclass for the future?